ti/instrumentation/utils.go

```diff
@@ -39,10 +39,11 @@ const (
 	outDir       = "%s/ti/callgraph/" // path passed as outDir in the config.ini file
 	tiConfigPath = ".ticonfig.yaml"
 	// Parallelism environment variables
-	harnessStepIndex  = "HARNESS_STEP_INDEX"
-	harnessStepTotal  = "HARNESS_STEP_TOTAL"
-	harnessStageIndex = "HARNESS_STAGE_INDEX"
-	harnessStageTotal = "HARNESS_STAGE_TOTAL"
+	harnessStepIndex    = "HARNESS_STEP_INDEX"
+	harnessStepTotal    = "HARNESS_STEP_TOTAL"
+	harnessStageIndex   = "HARNESS_STAGE_INDEX"
+	harnessStageTotal   = "HARNESS_STAGE_TOTAL"
+	HARNESS_REPORT_PATH = "harness_test_results.xml"
```

Comment:
Keep the constant in only place and import it everywhere.

Reply:
this is removed. Forgot to commit

Reply:
It's still here